android/koin-android/src/test/java/org/koin/test/android/ext/koin/KoinExtTest.kt

```diff
@@ -0,0 +1,168 @@
+package org.koin.test.android.ext.koin
+
+import android.app.Application
+import android.content.Context
+import io.mockk.every
+import io.mockk.mockk
+import java.lang.Exception
+import org.junit.Assert
+import org.junit.Test
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidFileProperties
+import org.koin.android.ext.koin.androidLogger
+import org.koin.android.logger.AndroidLogger
+import org.koin.core.KoinApplication
+import org.koin.core.annotation.KoinInternalApi
+import org.koin.core.logger.Level
+import org.koin.core.logger.Logger
+import org.koin.core.logger.MESSAGE
+import org.koin.core.registry.saveProperties
+
+class KoinExtTest {
+
+    @Test
+    fun `GIVEN KoinApplication and Debug level WHEN call android logger THEN setup logger with level expected`() {
+        Level.DEBUG.expetedFor { it.androidLogger(Level.DEBUG) }
+    }
+
+    @Test
+    fun `GIVEN KoinApplication and Error level WHEN call android logger THEN setup logger with level expected`() {
+        Level.ERROR.expetedFor { it.androidLogger(Level.ERROR) }
+    }
+
+    @Test
+    fun `GIVEN KoinApplication and None level WHEN call android logger THEN setup logger with level expected`() {
+        Level.NONE.expetedFor { it.androidLogger(Level.NONE) }
+    }
+
+    @Test
+    fun `GIVEN KoinApplication and Info level WHEN call android logger THEN setup logger with level expected`() {
+        Level.INFO.expetedFor { it.androidLogger(Level.INFO) }
+    }
+
+    @Test
+    fun `GIVEN KoinApplication and default level WHEN call android logger THEN setup logger with level expected`() {
+        Level.INFO.expetedFor { it.androidLogger() }
+    }
+
+    private fun Level.expetedFor(code: (KoinApplication) -> KoinApplication) {
+        val koinApplication = code(KoinApplication.init())
+
+        // THEN
+        Assert.assertTrue(koinApplication.koin.logger is AndroidLogger)
+        Assert.assertEquals(this, koinApplication.koin.logger.level)
+    }
+
+    @OptIn(KoinInternalApi::class)
+    @Test
+    fun `GIVEN KoinApplication and Application WHEN setup androidContext() THEN result as expected`() {
+        // GIVEN
+        val customLogger = object : Logger(Level.INFO) {
```

Comment:
I don't think we need to verify the logger here since we want to test the android context.